### PR TITLE
refactor: Pre-calculated constant for `convertLandauFwhmToGaussianSigma`

### PR DESCRIPTION
**Pull request by andiwand**
Pulled out of https://github.com/acts-project/acts/pull/2366 to pin down the hash and physmon diff

blocked by
- https://github.com/acts-project/acts/pull/2374

### CODE REVIEW

**Comment by felix-russo** [comment]
Any idea why we reconstruct less vertices now?
image

**Comment by andiwand** [comment]
not sure if we do - the plot is hard to read

these changes should not affect the AMVF results but do for unknown reasons. this is what I experienced a couple of times already. small changes in the sim or track fitting result in big changes in the AMVF hist cmp

I updated the reference because the problem seems unrelated

**Review by benjaminhuth — DISMISSED** [review]
Its a shame that things like `std::log` only get `constexpr` with C++26...
But looks good, let's get it in!

**Comment by paulgessinger** [comment]
physmon and the examples tests still seem to fail.

File: Core/src/Material/Interactions.cpp [hunk 1]
```diff
@@ -256,7 +256,8 @@ namespace {
 /// -> sigma = fwhm / (2 * sqrt(2 * log(2)))
 ///
 inline float convertLandauFwhmToGaussianSigma(float fwhm) {
-  return fwhm / (2 * std::sqrt(2 * std::log(2.0f)));
+  // return fwhm / (2 * std::sqrt(2 * std::log(2.0f)));
+  return fwhm * 0.42466090014400953f;
```
[the remark this answers is not in the document]
Comment (benjaminhuth) [reply Core/src/Material/Interactions.cpp:260]: feel free to ignore of course

**Comment by andiwand** [comment]
I am trying to find the problem in the AMVF which is causing this flip flop of vertices for small numerical changes. This diff here is quite nice to be able to find the spot

**Review by benjaminhuth — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results
### :red_circle: Some tests have failed!
Please investigate the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/6075332)!

|status|job|
|------|---|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864015) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsWorkflow.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864014) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateTracks.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864013) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsCoreSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864012) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864011) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864010) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateOrthogonalSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864009) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateClusters.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864008) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsPersistifyEDM.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864007) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsKfRefitting.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864006) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsExtrapolationAlgTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864005) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsITkTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864004) |
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864003) |
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864002) |
|:red_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864001) |
|:red_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31864000) |
|:red_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/31863999) |

**Comment by paulgessinger** [comment]
Athena changes did not come from this PR.

**Pull request merged**